src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_launcher_test.py

```diff
@@ -1283,7 +1292,8 @@ def test_oom_crash(self, mock_stdout):
           '/fake/corpus_oom'
       ], [
           '/fake/build_dir/fake_fuzzer', '-rss_limit_mb=2048', '-timeout=25',
```

Comment:
nit: can align to new line similar to others.

Reply:
yapf doesn't allow me to do this. Leaving as is.